documentation/modules/ROOT/pages/connectors/jdbc.adoc

```diff
@@ -1439,18 +1439,36 @@ To prevent the connector from rebalancing, set the total retry time (flush.retry
 |Property |Default |Description
 
 |[[jdbc-property-column-naming-strategy]]<<jdbc-property-column-naming-strategy, `+column.naming.strategy+`>>
-|`i.d.c.j.n.DefaultColumnNamingStrategy`
-|Specifies the fully-qualified class name of a `ColumnNamingStrategy` implementation that the connector uses to resolve column names from event field names. +
- +
-By default, the connector uses the field name as the column name.
-|[[jdbc-property-collection-naming-strategy]]<<jdbc-property-collection-naming-strategy, `+collection.naming.strategy+`>>
-|`i.d.c.j.n.DefaultCollectionNamingStrategy`
-|Specifies the fully-qualified class name of a `CollectionNamingStrategy` implementation that the connector uses to resolve table names from incoming event topic names. +
- +
-The default behavior is to: +
+|`io.debezium.connector.jdbc.naming.DefaultColumnNamingStrategy`
+|Specifies the fully-qualified class name of a `ColumnNamingStrategy` implementation that the connector uses to resolve column names from incoming event field names. +
++
+The default behavior is to use the field name as the column name without any transformation. +
++
+You can use the enhanced column naming capabilities by setting <<jdbc-property-column-naming-style-enabled, `+column.naming.style.enabled+`>> to `true`.
+
+|[[jdbc-property-column-naming-style-enabled]]<<jdbc-property-column-naming-style-enabled, `+column.naming.style.enabled+`>>
+|`false`
+|When set to `true`, enables enhanced column naming with support for naming styles, prefixes, and suffixes. When enabled, this activates the custom naming strategy which provides more flexible column name transformations.
 
-* Replace the `+${topic}+` placeholder in the xref:jdbc-property-collection-name-format[`collection.name.format`] configuration property with the event's topic.
-* Sanitize the table name by replacing dots (`.`) with underscores (`_`).
+|[[jdbc-property-column-naming-style]]<<jdbc-property-column-naming-style, `+column.naming.style+`>>
```

Comment:
We should revert these changes and consider adding a similar section to the JDBC sink's documentation like we have done for other connector-specific converters, see [here](https://debezium.io/documentation/reference/stable/connectors/mysql.html#debezium-mysql-connector-converters). This way we can document the new SMTs you've added, their configurations, and purposes.